sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -226,6 +228,11 @@ export { OperationOptions }
 // @public
 export type OperationOptionsBase = Pick<OperationOptions, "abortSignal" | "tracingOptions">;
 
+// @public
+export interface ParentSpanOptions {
```

Comment:
Do we have to expose this interface? Looks like it is used only in `TryAddOptions`. Why cant we inline this there like we do in Event Hubs?

Reply:
Okay, removed `ParentSpanOptions`.

There were both ParentSpanOptions and TryAddOptions in the event-hubs PR, the former was used for the getPartition, etc methods. I was under the impression that the separate interface would be helpful. But as I see those management methods were updated to use tracing options from the OperationOptions directly. So, all good with removing the `ParentSpanOptions` here.